src/process/command_exec.rs

```diff
@@ -34,7 +34,7 @@ pub fn decode_command(encoded_command: &str) -> String {
 
 pub fn format_powershell_command(command:String) -> String {
     format!(
-        "$ErrorActionPreference = 'Stop'; {} ; exit $LASTEXITCODE",
+        "[Console]::OutputEncoding = [System.Text.Encoding]::UTF8;$ErrorActionPreference = 'Stop'; {} ; exit $LASTEXITCODE",
```

Comment:
Thanks for powershell, have you tried with the 3 other executors (cmd, bash and sh) ? Because the issue was for cmd and psh and maybe it concerned other executors.

Reply:
I tried psh, bash and cmd. I will test it with sh.